### DISCUSSION TITLE
PowerPoint Morph Transition

### DISCUSSION THREAD

**Discussion by KevinNitroG** [YerongAI/Office-Tool#462] [Q&A] [answered]
Hello <3 I have installed office 365 from the app. But my PowerPoint doesn't have Morph Transition. The office also doesn't have new design either. I have installed the office and reinstall, but it stuck at 2%, disk ram cpu didn't go up. :(( What should I do. Thank you in advance!

**Comment by YerongAI** [comment, marked as answer]
First: make sure you have Microsoft 365 Apps for Enterprise (home edition is the same) installed.

> If your installation get into problems, try to remove Office from toolbox page, and verify your installation is good or not (installation management).

Second: clear the old activation (activate page) and reset Office settings to defaults (toolbox page).

Third: make sure you have valid license for activating 365. If not, use Office 2016 Mondo Volume license instead, and activate it through KMS. But you should know it's illegal.

**Reply by KevinNitroG** [reply]
Thank you for your help <3 it is so detail 🥰
